userinterface/map/routeplannermodule.cpp

```diff
@@ -366,3 +366,10 @@ void RoutePlannerModule::reverseCurrentRoute()
 
     emit requestRepaint();
 }
+
+void RoutePlannerModule::mergeWithCurrentRoute(int index)
```

Comment:
In my opinion, "merge" is too unspecific: will it get appended? prepended? points joined alternately? I would suggest to change every mention of "mergeWith" to "appendTo", because this is what the line below actually does.